### DISCUSSION TITLE
Аналог FarMenu.ini

### DISCUSSION THREAD

**Discussion by martid1703** [elfmz/far2l#1490] [Q&A] [answered]
Привет, подскажите как правильно сохранять пользовательское меню (F2)?
На Windows оно хранилось в FarMenu.ini, какой аналог на Linux и в какой директории его нужно хранить?
По F2->Ctrl+F4 видно, что файл создается в папке /tmp ?
![image](https://user-images.githubusercontent.com/31855565/215263573-8c09afd1-659b-4591-905a-79c13c2372db.png)

**Comment by unxed** [comment, marked as answer]
~/.config/far2l/settings/user_menu.ini

**Reply by martid1703** [reply]
спасибо!
